src/main/java/seedu/address/model/schedule/Schedule.java

```diff
@@ -0,0 +1,69 @@
+package seedu.address.model.schedule;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+
+public class Schedule {
+
+    private List<Event> events;
+
+    /**
+     * Every field must be present and not null.
+     */
+    public Schedule(List<Event> events) {
+        requireAllNonNull(events);
+        this.events = events;
+    }
+
+    public List<Event> getEvents() {
+        return Collections.unmodifiableList(events);
+    }
+
+    public void addEvent(Event event) {
+        events.add(event);
+    }
+
+    public void setEvents(List<Event> events) {
```

Comment:
Maybe include javadocs for this method. Or change the name of the method. Not quite sure what this does.